src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/catalog.cljs

```diff
@@ -143,17 +145,55 @@
                                               :value (get (:attributes @state) property-kwd)
                                               :onChange update-property
                                               :rows 3})
+                     (= typeahead "ontology")
+                     (style/create-text-field {:ref property-kwd
+                                               :className "typeahead"
+                                               :placeholder "Select an ontology."
```

Comment:
nit: this should be "select an ontology value" or "select a value". We're using the DOID ontology, so the end user can't actually choose a different ontology.